### ISSUE TITLE
Accessibility: Macos voiceover missing for UI

### ISSUE THREAD

**Issue by gaktive** [rancher-sandbox/rancher-desktop#2412]
### Actual Behavior

On macos, I installed RD 1.4.1 and turned on VoiceOver to see what descriptions are provided. The main feel is that with Electron, once you engage with an element, such as a button to bring up a new window, there's a visual indicator that shows the last item clicked but that means that the subsequent content is not read unless you do cursor focus elsewhere and go back into Rancher Desktop. Otherwise, there's a lot of "You are currently in a main, inside of web content" without actually reading the contents until the cursor focus resets.

Partial walkthrough:

- Post-install -- no description provided for the Configure PATH pop-up aside from Automatic & Manual radio buttons. Header with "There's one last step to finish updating Rancher Desktop" only mentions "Rancher Desktop".

- General -- only "web content" mentioned so none of the fields or checkboxes are vocalized/read, even the "Allow collection of anonymous statistics..."

- Application Settings -- header not mentioned but radio buttons Automatic & Manual are read. Checkbox for sudo access not read.

- Kubernetes Settings -- Settings not read, though it will read the radio buttons for toggling between containerd and moby. "Kubernetes version" and "Port" not read though once menu items selected, whatever you pick in the dropdown are read. When Enabling Kubernetes, System dialogue box is read. Any changes made to the services doesn't read the bottom left status as the virtual machine is stopped, started, restarted, etc. 

- Port Forwarding -- title mentioned but not the namespace table objects. Clicking Forward or Cancel will activate those buttons being read but the Local Port number isn't mentioned. 

- Images -- individual items in images table aren't read. Same for when using the vertical ellipsis to bring up the Scan option, though when the Scan details shows, there's a rectangle over the older Electron object which doesn't activate reading the Scan details. If you hop into another app and back in, Voiceover eventually reads the Scan details. 

- Troubleshooting -- title read but Logs and Factory Reset buttons only read; headers and content with radio buttons not read. 

### Steps to Reproduce

Turn Voiceover on the Mac and navigate through the UI

### Result

Partial readout of UI options unless cursor focus out/in RD.

### Expected Behavior

More information read from the verbiage in the app.

### Rancher Desktop Version

1.4.1

### Rancher Desktop K8s Version

n/a

### Which container runtime are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

Monterey 12.3.1

### What CPU architecture are you using?

arm64 (Apple Silicon)

**Comment by gaktive** [comment]
Some connection with https://github.com/rancher-sandbox/rancher-desktop/issues/2383
